src/service/performance-impl.js

```diff
@@ -301,8 +301,12 @@ export class Performance {
         recordedFirstInputDelay = true;
       } else if (entry.entryType === 'layoutJank') {
         this.aggregateJankScore_ += entry.fraction;
```

Comment:
Be aware that this will calculate layout shifts even for cases that involved user input. Is that what you want?

Reply:
I think this is okay for this case. Chrome 75 versions are turning down, and we'll soon have the user interaction-aware metrics in Chrome 77 in about a month.